apps/server/src/modules/task-card/controller/dto/task-card.params.ts

```diff
@@ -91,6 +70,12 @@ export class TaskCardParams {
 	})
 	courseId?: string;
 
+	@IsString()
+	@ApiProperty({
```

Comment:
adding `@MinLength(XYZ)` ? depending on validation

Reply:
also, sanitizeHTML() to strip away any html/js